### PR TITLE
Remove all references to Growth Stage

### PR DESCRIPTION
**Pull request by tobie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,9 +98,9 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
...
 
-#### Growth & At Large Project Representatives
+#### At Large Project Representatives
 
-According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth, At Large, and Incubating Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. From those nominees, two voting members are chosen via an [election process outlined in the CPC Governance](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#election-process-for-non-impact-voting-cpc-members).
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the At Large and Incubating Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. From those nominees, two voting members are chosen via an [election process outlined in the CPC Governance](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#election-process-for-non-impact-voting-cpc-members).
```
Comment (sendilkumarn) [inline README.md:103]: /master/HEAD/s

**Review by joesepi — APPROVED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (tobie) [reply README.md:103]: Heh, well of course this is going to conflict with #716.

**Review by eemeli — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -104,9 +98,9 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
...
 
-#### Growth & At Large Project Representatives
+#### At Large Project Representatives
 
-According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#voting-members), each of the Growth, At Large, and Incubating Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. From those nominees, two voting members are chosen via an [election process outlined in the CPC Governance](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#election-process-for-non-impact-voting-cpc-members).
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#voting-members), each of the At Large and Incubating Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. From those nominees, two voting members are chosen via an [election process outlined in the CPC Governance](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#election-process-for-non-impact-voting-cpc-members).
```
Comment (tobie) [inline README.md:103]:
```suggestion
According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#voting-members), each of the At Large and Incubating Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. From those nominees, two voting members are chosen via an [election process outlined in the CPC Governance](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#election-process-for-non-impact-voting-cpc-members).
```

File: README.md (hunk 1 above)
Comment (tobie) [reply README.md]: Fixed

**Review by ljharb — APPROVED** [review]

**Review by sendilkumarn — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**